CMakeLists.txt

```diff
@@ -144,6 +153,14 @@ export(TARGETS
   FILE BackscrubTargets.cmake
 )
 
+if(IWYU_ENABLE)
+  message(STATUS "IWYU path: ${IWYU_PATH}")
+  set_property(
+    TARGET backscrub deepseg videoio
+    PROPERTY CMAKE_CXX_INCLUDE_WHAT_YOU_USE "${IWYU_PATH};-Xiwyu;--no_fwd_decls;-Xiwyu;--mapping_file=${IWYU_MAPFILE}"
```

Comment:
This doesn't work for me - I need to set the property `CXX_INCLUDE_WHAT_YOU_USE` (without the `CMAKE_` prefix)

Reply:
Oops, misread the docs here. Should be fixed now.